server/parsec/components/postgresql/organization_update.py

```diff
@@ -38,6 +42,9 @@ def _q_update_factory(
         fields.append("user_profile_outsider_allowed = $user_profile_outsider_allowed")
     if with_minimum_archiving_period:
         fields.append("minimum_archiving_period = $minimum_archiving_period")
+    if with_tos:
+        fields.append("tos_updated_on = $tos_updated_on")
+        fields.append("tos_url = $tos_url")
```

Comment:
Shouldn't it be per locale?
```suggestion
        fields.append("tos_per_locale_urls = $tos_per_locale_urls")
```

Reply:
good catch !